### PR TITLE
Ui/wallet/data

### PR DESCRIPTION
**Pull request by PascalPrecht**

### CODE REVIEW

File: src/app/wallet/walletView.nim [hunk 1]
```diff
@@ -43,6 +47,12 @@ QtObject:
                           image: image))
     self.endInsertRows()
 
+  proc setDefaultAccount*(self: WalletView, account: string) =
+    self.defaultAccount = account
+
+  method getDefaultAccount*(self: WalletView): string {.slot.} =
+    return self.defaultAccount
```
Comment (PascalPrecht) [inline src/app/wallet/walletView.nim:54]: I noticed we've been using `proc` and `method` interchangeably. I guess both can be `{.slot.}` so maybe it makes sense to have some guide lines on when to use which?

File: src/app/wallet/walletView.nim [hunk 2]
```diff
@@ -61,6 +71,9 @@ QtObject:
     of AssetRoles.FiatValue: result = newQVariant(asset.fiatValue)
     of AssetRoles.Image: result = newQVariant(asset.image)
 
+  proc onSendTransaction*(self: WalletView, from_value: string, to: string, value: string, password: string): string {.slot.} =
+    result = self.sendTransaction(from_value, to, value, password)
```
Comment (PascalPrecht) [inline src/app/wallet/walletView.nim:75]: This one might as well be `method` ?

File: ui/app/AppLayouts/Wallet/WalletLayout.qml [hunk 3]
```diff
@@ -43,15 +43,66 @@ Item {
...
+                anchors.leftMargin: 24
+            }
+
+            TextField {
+                id: txtPassword
+                x: 19
+                y: 99
+                width: 239
+                height: 40
+                text: "0x2cd9bf92c5e20b1b410f5ace94d963a96e89156fbe65b70365e8596b37f1f165"
```
Comment (PascalPrecht) [inline ui/app/AppLayouts/Wallet/WalletLayout.qml:87]: The password value is the one that is hardcoded in `test.nim` as well. This is just a placeholder for now.

File: src/app/wallet/walletView.nim (hunk 1 above)
Comment (richard-ramos) [reply src/app/wallet/walletView.nim:54]: Found these interesting links

https://matthiashager.com/proc-method-nim
> In general, method should be used over proc when, and only when, you don't know at compile time which object type from your inheritance tree the function will be handling.

http://goran.krampe.se/2014/10/31/nim-and-oo-part-iii/
According to this, using methods incur in a small performance penalty, but really depends on what you are doing. For most real life code, that performance is not noticeable.

**Pull request merged**